applications/fake-services/replicated/kots-preflight.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: troubleshoot.sh/v1beta2
+kind: Preflight
+metadata:
+  name: preflight
+spec:
+  analyzers:
+    - clusterVersion:
+        outcomes:
+          - fail:
+              when: "< 1.16.0"
+              message: The application requires at least Kubernetes 1.16.0, and recommends 1.18.0.
+              uri: https://kubernetes.io
+          - warn:
+              when: "< 1.18.0"
+              message: Your cluster meets the minimum version of Kubernetes, but we recommend you update to 1.18.0 or later.
+              uri: https://kubernetes.io
+          - pass:
+              message: Your cluster meets the recommended and required versions of Kubernetes.
+    - distribution:
```

Comment:
It may need to add k0s to pass the embedded cluster preflight
```
          - pass:
              when: "== k0s"
              message: k0s is a supported distribution
```